### PR TITLE
docs: update juju tutorial to use version 3.0 instead of 2.9

### PR DESCRIPTION
**Pull request by jat-canonical**
This commit updates the docs for new users to start with juju 3.0 as that is the gold standard to use right with new features being developed as a priority for it.

### CODE REVIEW

File: howto/install/deploy-juju.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ Before you start the installation, ensure that you have the required credentials
 
 Juju is a tool for deploying, configuring and operating complex software on public or private clouds.
 
-To install Juju 2.9, enter the following command:
+To install Juju 3.0, enter the following command:
```
Comment (morphis) [inline howto/install/deploy-juju.md:30]: Should we say `3.x`?

**Review by morphis — CHANGES_REQUESTED** [review]

**Comment by keirthana** [comment]
There's also one more specific scenario of mentioning architecture for Juju 2.9.0 in that topic. Should we change that to align with Juju 3.0? See https://documentation.ubuntu.com/anbox-cloud/en/latest/howto/install/deploy-juju/#use-arm-instances

**Comment by jat-canonical** [comment]
> There's also one more specific scenario of mentioning architecture for Juju 2.9.0 in that topic. Should we change that to align with Juju 3.0? See https://documentation.ubuntu.com/anbox-cloud/en/latest/howto/install/deploy-juju/#use-arm-instances

I had intentionally left that in for people who might have a need to run 2.9 or 3.x as it was just extra information.

**Review by morphis — APPROVED** [review]

**Comment by keirthana** [comment]
> > There's also one more specific scenario of mentioning architecture for Juju 2.9.0 in that topic. Should we change that to align with Juju 3.0? See https://documentation.ubuntu.com/anbox-cloud/en/latest/howto/install/deploy-juju/#use-arm-instances
> 
> I had intentionally left that in for people who might have a need to run 2.9 or 3.x as it was just extra information.

Okay, if there will be some users who will want to stick with 2.9, it is worth keeping it. Thanks for clarifying

**Review by keirthana — APPROVED** [review]

**Pull request merged**